packages/react-form-state/src/components/List.tsx

```diff
@@ -53,6 +55,7 @@ export default class List<Fields> extends React.Component<
             initialValue: initialFieldValue,
             dirty: value !== initialFieldValue,
             error: get(error, [index, fieldPath]),
+            disabled,
```

Comment:
This seems a bit weird to me but I don't want to change `disabled` to take an object with the fieldPath... Basically this means that when the disabled prop gets set on the parent, it'll always disable all the children. You can't selectively disable children unless you manage that state yourself...

This is fine for the `disableOnSubmit` feature... but I just want to make sure this isn't going to be a problem later on.

cc @michelleyschen @TheMallen

Reply:
I think this is fine for now, since we have things like `dirty` that also has value instead of object.
Defin' something we want to change for v2 thou.

Reply:
Yeah this is only a problem if we update it selectively for some, which we don't do now. Michelle is right that we need to change a lot of how nested stuff works in the future anyway.